tests/album-create.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+  let artistid;
+  beforeEach(async () => {
+    const { status, body } = await request(app)
+      .post('/artists')
+      .send({ name: 'Tame Impala', genre: 'rock' });
+    
+    expect(status).to.equal(201);
+    expect(body.name).to.equal('Tame Impala');
+    expect(body.genre).to.equal('rock');
+    artistid = body.id
+  });
+     
+
```

Comment:
Never need more than one consecutive blank line in your code, you should one of these